rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -1602,9 +1602,9 @@ def test_remove_from_aave_pool(ethereum_transaction_decoder, ethereum_accounts,
             event_type=HistoryEventType.SPEND,
             event_subtype=HistoryEventSubType.RETURN_WRAPPED,
             asset=Asset('eip155:1/erc20:0xFd2a8fA60Abd58Efe3EeE34dd494cD491dC14900'),
-            balance=Balance(amount=FVal('2619.297635390037009904')),
+            balance=Balance(amount=FVal('2904.458947543711333839')),
```

Comment:
Why did you change this test?

As I told you the changes you made are touching code they should not be (v2). This is wrong.

You are swapping a3cRV which is the v2 aToken returned here for the aDAI amount which is part of the same transaction.

https://etherscan.io/tx/0xb0a45bc41a83b2bdf2e06b9913a2e4c8b0d7f3080030807a0a06f301287424e9